WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -47,15 +50,15 @@ struct PointOfSaleDashboardView: View {
             .padding(.bottom, Constants.floatingControlBottomPadding)
             .trackSize(size: $floatingSize)
             .accessibilitySortPriority(1)
-            .renderedIf(posModel.itemsViewState.containerState != .loading)
+            .renderedIf(posModel.currentViewState.containerState != .loading)
```

Comment:
This took me a while: By leaving this one as `posModel.itemViewstate`, when switching to Coupons we were always stuck in .loading state, not showing coupons correctly. Of course the compiler didn't complain, but was tricky to find why the view state was wrong. I think we can move those ones to some computed property, or grab them directly from reading a variable from the aggregate model.